tests/fixtures/rules/indent/indent-invalid-fixture-1.js

```diff
@@ -135,8 +135,8 @@ switch (a) {
 }
 
 a.b('hi')
-   .c(a.b()) // NO ERROR: DON'T VALIDATE MULTILINE STATEMENTS
-   .d(); // NO ERROR: DON'T VALIDATE MULTILINE STATEMENTS
+   .c(a.b()) // <-
```

Comment:
It is indented three spaces but the lint rule is for two spaces. All lines with indentation errors in this file are identified with a comment with `<-` or `->` indicating whether the line indentation should be moved left or right (respectively) to comply with the linter.